pallets/randomness/src/weights.rs

```diff
@@ -112,6 +114,17 @@ impl<T: frame_system::Config> WeightInfo for SubstrateWeight<T> {
 			.saturating_add(T::DbWeight::get().reads(4 as Weight))
 			.saturating_add(T::DbWeight::get().writes(4 as Weight))
 	}
+	// Storage: Randomness NotFirstBlock (r:1 w:0)
+	// Storage: System Digest (r:1 w:0)
+	// Storage: AuthorMapping MappingWithDeposit (r:1 w:0)
+	// Storage: Randomness LocalVrfOutput (r:1 w:1)
+	// Storage: Randomness RandomnessResults (r:1 w:0)
```

Comment:
```suggestion
	// Storage: Randomness RandomnessResults (r:1 w:1)
```
Forgot to insert a request first, fixing now